av1-rtp-spec.md

```diff
@@ -252,50 +247,32 @@ taking two bytes for the first and second OBU and one byte for the last (N) OBU.
 Whether or not the first and/or last OBU is fragmented is signaled in the
 aggregation header.
 
-**TODO:** Add a paragraph that describes how a temporal unit is mapped into RTP
-packets. I think a new temporal unit should start a new RTP packet. You skip the
-temporal delimiter OBU, and then packetize the remaining OBUs in one or more
-packets, with fragmentation as needed.
-{:.alert .alert-danger }
-
-* * *
-
-<div class="alert alert-danger" markdown="1">
-
-2019/01/15 Notes:
-
-A packet must not include OBUs across a TD
-All OBUs in a packet must have the same temporal_id and spatial_id
-OBUs with layering information must not be aggregated with OBUs that don't have layering information (layering information=extension header).
-If sequence header is present, it should (not must) be the first OBU in a packet.
-Q: May not be needed.
-A packet cannot include OBUs from different frames.
-Q: for hidden frames, the various frames may all be required anyway, so maybe we allow aggregation of frames with associated hidden frames (of the same temporal unit).
+## 5. Packetization Rules
 
-Q: Should we support tile list OBUs?
+Each RTP packet MUST contain OBUs that belong to a single temporal unit.
 
-Packetization Exercises:
+The temporal delimiter OBU, if present, SHOULD be removed when transmitting, and
+MUST be ignored by receivers.
 
-TD   SH MD MD(0,0) FH(0,0) TG0(0,0) MD(0,1) FH(0,1) TG0(0,1) ...
+With the exception of sequence header and metadata OBUs, an RTP packet MUST

```

Comment:
Does this rule follow from combining rules at line#252 (single temporal unit) and rule at line#262 (temporal id and spatial id must be same when present)?